CHANGELOG.md

```diff
@@ -1,18 +1,16 @@
 # Changelog
-
-- Friday \*\* June 2022
-  v0.16.2
-
-## Enhancements
-
-- We introduced a new function called `set_assume_org_id` for incoming super organization feature. By using this method, you can assume the id of a child organization as their super organization, and make your calls on behalf of them.
-
-A super organization is loosely modelled on superuser. So you can as a company ACME have specific criteria and permissions - and then you can share these with your child organisations. This allows you fine grained control of users and their roles and permissions and the ability to share settings and voices across orgs. We were informed by IAM from AWS in our design. If you want access let us know, we're working hard on this. This is part of a whole 
-
-- *Voices* We have some great partnerships coming up with 2 new voice providers. Reach out to us if you want to know more :) 
-## Coming soon 
-Under msnr we will soon have another german voice. We're testing this with some beta customers the `margareta-v1` voice. 
-
+- Friday 26th June 2022
+v.0.16.3 
+## SuperOrg
+- Listing superorgs - the abiltiy to list these organisations. `list_org_id` 
```

Comment:
id remove `list_org_id` as wearent sure what to call it atm

Reply:
What are we calling them?

Reply:
its not added to sdks yet, i should create a ticket for @guillem-211